velox/functions/lib/tests/SliceTest.cpp

```diff
@@ -126,6 +128,36 @@ TEST_F(SliceTest, prestoTestCases) {
   }
 }
 
+TEST_F(SliceTest, integerStartAndLength) {
```

Comment:
Is this a copy-paste from some other test? It would be nice to refactor to avoid code duplication.